config/qt5_core.mpb

```diff
@@ -0,0 +1,56 @@
+project {
+  expand(QT5_INCDIR) {
+    $QT5_INCDIR
+    $(QT5_DIR)/include
+  }
+
+  requires += qt5
+  includes += $(QT5_INCDIR)
+  libpaths += $(QT5_DIR)/lib $(QT5_DIR)/bin
```

Comment:
Does /bin need to be here?  That would mean there's a case where the linker needs to find a file in $(QT5_DIR)/bin.

Reply:
No that was an attempt to fix the dlls not being found on Windows, I can remove it.